hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -827,8 +827,8 @@ def create_volume(self, volume):
                 extras['snapCPG'] = cpg
                 volume['snap_cpg'] = cpg
 
-            # Only set the dedup option if the backend supports it.
-        if self.API_VERSION >= DEDUP_API_VERSION:
+        # Only set the dedup option if the backend supports it.
+        if self.API_VERSION >= DEDUP_API_VERSION and tdvv:
```

Comment:
Requested @imran-ansari , to try and test volume creation on 3.2.2/3.3.1 array to see if there is any impact on older firmwares.